player/lua/ytdl_hook.lua

```diff
@@ -320,7 +320,8 @@ local function add_single_video(json)
 
             if not (sub_info.data == nil) then
                 sub = "memory://"..sub_info.data
-            elseif not (sub_info.url == nil) then
+            elseif not (sub_info.url == nil) and
+                url_is_safe(sub_info.url) then
```

Comment:
Should this have been in the point release? Is this really just a nit?

Reply:
Well, it's still somewhere where youtube-dl could give us unsafe urls, so I guess it could've been.

Would need something like
```html
<video id="video" controls preload="metadata">
   <source src="video/video.mp4" type="video/mp4">
   <source src="video/video.webm" type="video/webm">
   <track label="English" kind="subtitles" srclang="en" src="unsafe://stuff" default>
</video>
```